packages/@o3r/core/schematics/rule-factories/monorepo/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Rule, SchematicContext, SchematicsException, Tree } from '@angular-devkit/schematics';
+import * as commentJson from 'comment-json';
+import { PackageJson } from 'type-fest';
+
+/**
+ * Update root package.json to include workspaces
+ *
+ * @param patterns workspaces patterns to include in package.json
+ */
+export function addWorkspacesToProject(patterns = ['apps/*', 'libs/*']): Rule {
```

Comment:
Maybe they should be set in constants (as you need to add them in `application`, `sdk` and `library` config in the angular.json)